crates/analytics/docs/clickhouse/scripts/authentications.sql

```diff
@@ -167,7 +170,8 @@ SELECT
     ds_trans_id,
     directory_server_id,
     acquirer_country_code,
-    sign_flag
+    sign_flag,
+    organization_id
```

Comment:
```suggestion
    organization_id,
    sign_flag
```

order strictly matters between the SELECT statement and the target table